src/drivers/driver.ts

```diff
@@ -1095,6 +1104,8 @@ export abstract class CMakeDriver implements vscode.Disposable {
   }
 
   async configure(trigger: ConfigureTrigger, extra_args: string[], consumer?: proc.OutputConsumer, withoutCmakeSettings: boolean = false): Promise<number> {
+    // Check if this is an automatic configuration and adjust the logging messages based on that.
```

Comment:
I struggle with the term "automatic configuration".  To me this doesn't suggest that "CMake wasn't configured." I would prefer that we rename this to something else.  "configureFromCache" or "useCachedConfiguration" are some ideas.

Reply:
changed to `readCachedConfiguration`

Reply:
Can you rename the variable too?

Reply:
I renamed the variables as well